kirun-js/src/engine/function/system/array/Delete.ts

```diff
@@ -59,11 +55,9 @@ export class Delete extends AbstractArrayFunction {
         }
 
         if (index != -1) {
-            for (let i = index; i <= deletableSize; i++) {
-                source.remove(index);
-            }
+            source.splice(index, deletableSize);
         }
 
-        return new FunctionOutput(List.of(EventResult.outputOf(Map.of())));
+        return new FunctionOutput([EventResult.outputOf(new Map([]))]);
     }
 }
```

Comment:
Instead of deleting a sub array. Please change the logic to delete one or more elements from an array. Change the parameter to vargs so it can support one or many.